cqrs-codegen/impl/src/util.rs

```diff
@@ -30,6 +30,7 @@ where
 
 /// Checks that no attribute with a given `attr_name` exists.
 /// Returns error if found.
+#[allow(dead_code)]
```

Comment:
I think this function may still be useful later, for other proc-macro implementations, so I've marked it with `#[allow(dead_code)]`.